static/app/views/explore/spans/spansTab.tsx

```diff
@@ -222,40 +214,31 @@ export function SpansTabContentImpl({
             })}
           />
         </StyledPageFilterBar>
-        {dataset === DiscoverDatasets.SPANS_INDEXED ? (
-          <SpanSearchQueryBuilder
-            projects={selection.projects}
-            initialQuery={query}
-            onSearch={setQuery}
-            searchSource="explore"
-          />
-        ) : (
```

Comment:
I've kind of forgotten, what do users see today that aren't on EAP EA? Do they see the spans indexed components as the default and would this remove the search bar for them

Reply:
EAP RPC is the default dataset for everyone and only a few people had access to switch to other datasets. So this is a no-op to most users.